client/extensions/woocommerce/state/ui/shipping/zones/methods/actions.js

```diff
@@ -33,32 +64,29 @@ export const removeMethodFromShippingZone = ( siteId, methodId ) => {
  * Changes the given shipping method into a shipping method of a new type. Note that, underneath, this removes the
  * shipping method from the shipping zone, and creates a new one of the given type.
  * @param {Number} siteId Site ID.
- * @param {String} methodId ID of the shipping method to change type.
  * @param {String} methodType New shipping method type to switch to. For example, "free_shipping", "local_pickup".
  * @return {Object} Action object.
  */
-export const changeShippingZoneMethodType = ( siteId, methodId, methodType ) => {
-	return { type: WOOCOMMERCE_SHIPPING_ZONE_METHOD_CHANGE_TYPE, siteId, methodId, methodType };
+export const changeShippingZoneMethodType = ( siteId, methodType ) => {
+	return { type: WOOCOMMERCE_SHIPPING_ZONE_METHOD_CHANGE_TYPE, siteId, methodType };
 };
 
 /**
  * Changes the title of the given shipping method.
  * @param {Number} siteId Site ID.
- * @param {String} methodId ID of the shipping method.
  * @param {String} title New user-facing title for the shipping method.
  * @return {Object} Action object.
  */
-export const changeShippingZoneMethodTitle = ( siteId, methodId, title ) => {
-	return { type: WOOCOMMERCE_SHIPPING_ZONE_METHOD_EDIT_TITLE, siteId, methodId, title };
+export const changeShippingZoneMethodTitle = ( siteId, title ) => {
+	return { type: WOOCOMMERCE_SHIPPING_ZONE_METHOD_EDIT_TITLE, siteId, title };
 };
 
 /**
  * Enables or disables the given shipping zone method.
  * @param {Number} siteId Site ID.
- * @param {String} methodId ID of the shipping method.
  * @param {Boolean} enabled Whether to enable (true) or disable (false) the shipping method.
  * @return {Object} Action object.
  */
-export const toggleShippingZoneMethodEnabled = ( siteId, methodId, enabled ) => {
```

Comment:
Since you can enable/disable a method from the list (without opening the modal), you need to keep the old action signature (specify the `methodId`)